release/k3s/k3s.go

```diff
@@ -181,6 +181,8 @@ func (r *Release) SetupK8sRemotes(_ context.Context, ghClient *github.Client) er
 		}
 	}
 
+
```

Comment:
Is your editor configured to format your files?

Reply:
No, but I tried using lunarvim and it probably did it.